### ISSUE TITLE
Some questions about train

### ISSUE THREAD

**Issue by LiangSiyuan21** [kuangliu/torchcv#45]
When I train the VOC07+12 datasets and test the VOC07, I run train.py correctly.But after about 40 epoches, my train loss is declining, But THE TEST LOSS is rising. I train the datasets in original args.But I encountered the PROBLME which may be overfitting.
